### ISSUE TITLE
Add forced HTTPS support to obofoundry.org using GitHub pages; cleanup

### ISSUE THREAD

**Issue by nlharris** [OBOFoundry/OBOFoundry.github.io#1674]
http://obofoundry.org works; https just spins. @ddooley noted, "It would be web-consistent to put ssl cert on there." @cmungall said, "We should definitely do this.. we are stuck in a weird space as the URL is partly owned by Barry but haven’t been able to transfer it over (not due to any desire not to relinquish, it’s a technical thing)."

**Comment by cmungall** [comment]
@kltm do we have a ticket somewhere else for transferring obofoundry.org DNS entry from Barry to route53?

once this is done https support is trivial we just map this directly to github pages as we do for other sites

**Comment by cmungall** [comment]
This is the ticket for the handover: #393

**Comment by nlharris** [comment]
@cmungall are you sure that's the right ticket? it doesn't seem obviously related.

**Comment by kltm** [comment]
Hm. While it would be good to transfer the domain's ownership for a lot of reasons, I don't believe that getting HTTPS/TLS working is related. The server that hosts the site should be otherwise under independent control.

**Comment by kltm** [comment]
Clarifying the actions here:

- [ ] getting control of the domain
- [ ] directing to github
- [ ] get rid of the AWS EC2 proxy
- [ ] enabling https with GH domain controls

**Comment by kltm** [comment]
Also see https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1204 and

**Comment by kltm** [comment]
Assuming that what is desired is a forced for all traffic (i.e. all http -> https).
@cmungall @nlharris

**Comment by kltm** [comment]
@cmungall Noting that A record TTL in Route 53 is already low (300), so we can commence experiments with minimal downtime whenever we want, assuming that the proxypass `/ontologies -> http://www.berkeleybop.org/ontologies` is no longer needed.

**Comment by cmungall** [comment]
let's do it, we don't need the forward

**Comment by kltm** [comment]
Okay, I believe I've switched over the domain A record to be directly pointed at GH servers after setting up the domain in Pages, following  https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain
HTTPS does not yet seem to be available as an option; maybe need to wait a little:
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/troubleshooting-custom-domains-and-github-pages#https-errors
If anybody is seeing issues or regressions, please let me know ASAP.

**Comment by nlharris** [comment]
https://obofoundry.org/ gave a warning, but when I said "do it anyway", it worked! Thank you, Seth! 🙌

**Comment by kltm** [comment]
It is not forced yet, as some of the cert stuff is apparently in process, but forced upgrade should be available soon.

**Comment by kltm** [comment]
@cmungall @nlharris I've now turned on enforcement and it seems to work after a little testing. I will keep the old instance up as a backup for a little, but have deactivated the apache server to test what we have. If we don't run into any problems, I'll bring down and destroy the instance, completing the final item on the list https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1674#issuecomment-984035637

**Comment by ddooley** [comment]
works for me flawlessly now, from http:// to https:// ... glad that worked so smoothly, and a fast issue to resolve!

**Comment by nlharris** [comment]
Ha, maybe it appeared fast, but you didn't see all the work @kltm had to do behind the scenes!

**Comment by nlharris** [comment]
@kltm no pressure just wondering if there's any update on this

**Comment by kltm** [comment]
@nlharris The action list is here https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1674#issuecomment-984035637
The single remaining item is destroying the VM. As it does no harm and acts a a backup, it's has been pushed down the list of priorities for the moment.

**Comment by kltm** [comment]
This machine has not been `stopped`; things look fine. Looking at all the other things we have stopped but not terminated, we probably need to do a general cleaning. That said, I think this is far enough along to close.
